stable/storageos-operator/README.md

```diff
@@ -31,6 +33,8 @@ additional capacity, features and support plans contact sales@storageos.com.
 * High Availability - synchronous replication insulates you from node failure.
 * Delta Sync - replicas out of sync due to transient failures only transfer
     changed blocks.
+* Multiple AccessModes - dynamically provision ReadWriteOnce or ReadWriteMany
+  volumes
```

Comment:
It doesn't matters but please add one more level of indentation to maintain consistency with other points.